README.rst

```diff
@@ -4,18 +4,20 @@ CPython CMake Buildsystem
 Overview
 --------
 
-This is a replacement buildsystem for CPython.  The existing (autotools based)
-buildsystem for CPython is overcomplicated and makes cross-compiling very
-difficult.
+A replacement buildsystem for CPython.
 
-This cmake-based buildsystem is better because:
+This `CMake <http://cmake.org>`_ buildsystem has the following advantages:
 
+* No compiled program for the target architecture is used in the build
+  itself.  This makes *cross-compiling* very well supported.
+* Same build information for all platforms - there's no need to maintain the
+  autotools configuration separately from four different MSVC project files.
+* Support for other build system generators like `Ninja
+  <https://martine.github.io/ninja/>`_, `Sublime Text
```

Comment:
May be a a line talking about IDE support would be better here ? Sublime is not a build system.